src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/RazorCompletionContext.cs

```diff
@@ -2,10 +2,13 @@
 // Licensed under the MIT license. See License.txt in the project root for license information.
```

Comment:
This file is the root of all changes in this PR. Basically we now add absolute index and owners and then remove unneeded `SourceSpan`'s from consumers because info is included here now.